WebUI build failed with VueJS >= 3.3.0

After merging the PR https://github.com/nicolargo/glances/pull/2388 Glances WebUI failed to build:

```
> build
> webpack --progress --mode=production

[webpack-cli] Failed to load '/home/nicolargo/dev/glances/glances/outputs/static/webpack.config.js' config
[webpack-cli] Error: @vitejs/plugin-vue requires vue (>=3.2.13) or @vue/compiler-sfc to be present in the dependency tree.
    at Object.<anonymous> (/home/nicolargo/dev/glances/glances/outputs/static/node_modules/vue-loader/dist/compiler.js:14:15)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/nicolargo/dev/glances/glances/outputs/static/node_modules/vue-loader/dist/index.js:29:20)
    at Module._compile (internal/modules/cjs/loader.js:999:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
make: *** [Makefile:117: webui] Error 2
```

Problem is reproduced with all VueJS version >= 3.3.0. 

Downgraded to VueJS version 3.2.47 solves the issue.

```
package.json:        "vue": "^3.2.47"
```

--- a/glances/outputs/static/package.json
+++ b/glances/outputs/static/package.json
@@ -9,22 +9,23 @@
         "vue": "^3.2.47"
     },
     "devDependencies": {
+        "@vue/compiler-sfc": "^3.3.2",
         "copy-webpack-plugin": "^11.0.0",
         "css-loader": "^6.7.3",
         "del": "^7.0.0",
-        "eslint": "^8.32.0",
-        "eslint-plugin-vue": "^9.9.0",
-        "html-webpack-plugin": "^5.5.0",
+        "eslint": "^8.40.0",
+        "eslint-plugin-vue": "^9.12.0",
+        "html-webpack-plugin": "^5.5.1",
         "less": "^4.1.3",
         "less-loader": "^11.1.0",
-        "sass": "^1.57.1",
-        "sass-loader": "^13.2.0",
-        "style-loader": "^3.3.1",
+        "sass": "^1.62.1",
+        "sass-loader": "^13.2.2",
+        "style-loader": "^3.3.2",
         "url-loader": "^4.1.1",
-        "vue-loader": "^17.0.1",
-        "webpack": "^5.75.0",
-        "webpack-cli": "^5.0.1",
-        "webpack-dev-server": "^4.11.1"
+        "vue-loader": "^17.1.1",
+        "webpack": "^5.82.1",
+        "webpack-cli": "^5.1.1",
+        "webpack-dev-server": "^4.15.0"
     },
     "scripts": {
         "build": "webpack --progress --mode=production",

--- a/glances/outputs/static/package.json
+++ b/glances/outputs/static/package.json
@@ -3,10 +3,10 @@
     "dependencies": {
         "bootstrap": "^3.4.1",
         "favico.js": "^0.3.10",
-        "hotkeys-js": "^3.10.1",
+        "hotkeys-js": "^3.10.2",
         "lodash": "^4.17.21",
-        "sanitize-html": "^2.8.1",
-        "vue": "^3.2.47"
+        "sanitize-html": "^2.10.0",
+        "vue": "^3.3.2"
     },
     "devDependencies": {
         "@vue/compiler-sfc": "^3.3.2",

